llama-index-integrations/vector_stores/llama-index-vector-stores-lindorm/pyproject.toml

```diff
@@ -0,0 +1,66 @@
+[build-system]
+build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core"]
+
+[tool.codespell]
+check-filenames = true
+check-hidden = true
+skip = "*.csv,*.html,*.json,*.jsonl,*.pdf,*.txt,*.ipynb"
+
+[tool.llamahub]
+contains_example = false
+import_path = "llama_index.vector_stores.lindorm"
+
+[tool.llamahub.class_authors]
+LindormVectorStore = "aliyun"
+
+[tool.mypy]
+disallow_untyped_defs = true
+exclude = ["_static", "build", "examples", "notebooks", "venv"]
+ignore_missing_imports = true
+python_version = "3.8"
+
+[tool.poetry]
+authors = ["Your Name <you@example.com>"]
+description = "llama-index vector_stores lindorm integration"
+exclude = ["**/BUILD"]
+license = "MIT"
+name = "llama-index-vector-stores-lindorm"
+readme = "README.md"
+version = "0.1.0"
+
+[tool.poetry.dependencies]
+python = ">=3.8.1,<4.0"
+llama-index-core = "^0.10.1"
```

Comment:
We can add opensearch-py as a dependency here (and anything else I missed!)

Reply:
Thanks for your reminder! 😊 I have added `opensearch-py` and ensured it follows our project's style conventions. Here is the updated configuration in `pyproject.toml`:

```toml
[tool.poetry.dependencies]
python = ">=3.8.1,<4.0"
llama-index-core = "^0.10.1"
opensearch-py = { version = "^2.4.2", extras = ["async"] }

Reply:
@logan-markewich  Hi Logan, could you please trigger the workflows and review the changes when you have a moment? Thanks! 😊